jetty-client/src/main/java/org/eclipse/jetty/client/http/HttpChannelOverHTTP.java

```diff
@@ -39,6 +40,8 @@
     private final HttpConnectionOverHTTP connection;
     private final HttpSenderOverHTTP sender;
     private final HttpReceiverOverHTTP receiver;
+    private final AtomicLong _inMessages = new AtomicLong();
```

Comment:
Do we need longs here?  but on the other hand, ints any faster/better on modern CPUs?

Reply:
Actually it should probably be a LongAdder rather than an Atomic

Reply:
changed to LongAdder.